.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,6 +3,7 @@ This pull-request
 - [ ] resolves #<issue-id>
 - [ ] has a meaningful title
 - [ ] complies to the [code of conduct](../blob/master/CODE_OF_CONDUCT.md)
+- [ ] has a Changelog entry
```

Comment:
Does this mean the **changes** part below or somewhere else? I wouldn't want to check an extra checkbox for that every time, as I already have to edit it

Reply:
Maybe we should then have two things:
`- [ ] contains a breaking change, which is documented in the CHANGELOG.md`
`- [ ] contains a new feature, which is documented in the CHANGELOG.md`

Reply:
that's a good approach

Reply:
Approved if you change it :)